### PR TITLE
#111 Fix `Args` in Scala 3

### LINKED ISSUE #111: Why doesn't `sourcecode.Args` capture argument values under Scala 3?
Given this method:
```
  def printArgs(a: Int, b: Long, c: String): Unit = {
    val args = implicitly[sourcecode.Args]
    println(args)
  }
```
When I call `printArgs(1, 2, "3")`, the following lines are printed to the console:

**Scala 2.13.5**
```
Args(List(List(Text(1,a), Text(2,b), Text(3,c))))
```

**Scala 3.0.0-RC3**
```
Args(List(List(Text(None,a), Text(None,b), Text(None,c))))
```
I vaguely remember reading somewhere that this is a shortcoming of Scala 3's metaprogramming. Is this correct?

### PR DESCRIPTION
**Pull request by arainko**
Fixes #111 

This PR fixes the behavior of `Args` in Scala 3 - the values are now actually there.

### CODE REVIEW

**Review by lefou — APPROVED** [review]
Looks good to me. Thanks!

**Comment by joan38** [comment]
Hi , I'm impacted by this.
Is there a release of this?

**Comment by joan38** [comment]
Sounds like not. @lefou can we make a release? The last one was over a year ago

**Comment by lefou** [comment]
@joan38 If no maintainer will do it, I can squeeze it into my FOSS todo list. ;)

**Comment by joan38** [comment]
It would be nice please as we are blocked on this to upgrade to Scala 3.
It should be a matter of pushing a new tag right?

**Comment by lefou** [comment]
> It would be nice please as we are blocked on this to upgrade to Scala 3. 

Done.

> It should be a matter of pushing a new tag right?

Technically yes, but no. Cutting a release is more than "just" creating a tag, at least if you have more than three users and care about them. It's also editing a changelog, reviewing stuff etc.

**Comment by joan38** [comment]
Ah yes this is true!
Thanks @lefou greatly appreciated

**Pull request merged**
